src/settings/settingsdialog.cpp

```diff
@@ -105,6 +109,10 @@ SettingsDialog::SettingsDialog(MainWindow *parent)
 
     ui->ocrLanguagesListWidget->addLanguages(parent->ocr()->availableLanguages());
 
+    for (int i = 1; i <= QOnlineTranslator::Zulu; ++i)
+        if (!QOnlineTts::validRegions(static_cast<QOnlineTranslator::Language>(i)).isEmpty())
+            ui->googleLanguageComboBox->addItem(QOnlineTranslator::languageName(static_cast<QOnlineTranslator::Language>(i)), i);
```

Comment:
I think you can iterate over `regions` here directly.

Reply:
Sorry, I don't understand. Here I'm filling the language combo box with languages that has at least one valid region option. What can I do with `QOnlineTts::regions`?

Reply:
Sorry, I made a wrong assumption. Explained it upper in a similar suggestion.